dataclients/kubernetes/definitions/ingressvalidator.go

```diff
@@ -5,49 +5,87 @@ import (
 	"fmt"
 
 	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/metrics"
+	"github.com/zalando/skipper/routing"
 )
 
-type IngressV1Validator struct{}
+type IngressV1Validator struct {
+	FilterRegistry          filters.Registry
+	PredicateSpecs          []routing.PredicateSpec
+	Metrics                 metrics.Metrics
+	EnableWebhookValidation bool
```

Comment:
Better rename to `EnableAdvancedValidation`, this naming implies it enables the webhook itself which is not true